charts/presidio/values.yaml

```diff
@@ -38,6 +38,9 @@ api:
 analyzer:
   name: presidio-analyzer
   imagePullPolicy: Always
+  textAnalyticsKey: <key>
+  textAnalyticsEndpoint: <endpoint>
+  textAnalyticsApiPath: /text/analytics/v3.0-preview.1/entities/recognition/pii
```

Comment:
not sure this is a parameter

Reply:
The TA API is still in preview, so we want to be able to change the endpoint when it goes GA. Will this help?

Reply:
the question is WHO will update this value. the user? think not. so it will be a new presidio version.
in any case, having it in the values files means re-installing presidio.

bottom line, I think this should be removed.
@itye-msft for your consideration.